### PR TITLE
docs for unreleased/unmerged zipkin functionality

### PR DESCRIPTION
**Pull request by christian-posta**
@rshriram would love thoughts on this..

one section where I wanted to make it easy for a user to install `istio.yaml` with the zipkin images, but didn't want to copy/paste directly into the doc (as it's quite verbose) and just linked to a gist.. happy to move that gist to istio repo or explore alternatives.

### CODE REVIEW

File: _docs/tasks/zipkin-tracing.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: Collecting Zipkin Trace Spans
```
Comment (rshriram) [inline _docs/tasks/zipkin-tracing.md]: Distributed Request Tracing

File: _docs/tasks/zipkin-tracing.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+overview: How to configure the proxies to send tracing spans to Zipkin
+
```
Comment (rshriram) [inline _docs/tasks/zipkin-tracing.md]: s/spans/requests.

File: _docs/tasks/zipkin-tracing.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+![Zipkin Istio Dashboard](./img/zipkin_dashboard.png)
+
+If you click into a trace you should see the following:
+
+![Zipkin Istio Dashboard](./img/zipkin_span.png)
+
+Do note that the service names are not populated correctly... yet.
+
+## What's next
```
Comment (rshriram) [inline _docs/tasks/zipkin-tracing.md]: How about a Understanding What Happened section, where we explain how traces are collected, and then call out the fact that tracing is not free lunch, the user has to propagate zipkin trace headers (list the headers here).
```
X-Ot-Span-Context
X-Request-Id
X-B3-TraceId
X-B3-SpanId
X-B3-ParentSpanId
X-B3-Sampled
X-B3-Flags
```

**Comment by christian-posta** [comment]
Thx @rshriram updated the PR.

File: _docs/tasks/zipkin-tracing.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+    for ihdr in incoming_headers:
+        val = request.headers.get(ihdr)
+        if val is not None:
+            headers[ihdr] = val
+            #print "incoming: "+ihdr+":"+val
+
+    return headers
+```
+
```
Comment (rshriram) [inline _docs/tasks/zipkin-tracing.md]: nice!

**Pull request merged**